pom.xml

```diff
@@ -273,6 +274,11 @@
     <build>
         <pluginManagement>
             <plugins>
+                <plugin>
+                    <groupId>org.sonatype.central</groupId>
+                    <artifactId>central-publishing-maven-plugin</artifactId>
+                    <version>${central.publishing.plugin.version}</version>
```

Comment:
ditto, I believe that we do not need this `pluginManagement` section, just define the version in the plugins section.